### PR TITLE
Adjust pjax block for contact import block

### PR DESCRIPTION
**Pull request by norkans7**
Fixes https://github.com/rapidpro/rapidpro/issues/1326

### CODE REVIEW

File: templates/contacts/contact_import.haml [hunk 1]
```diff
@@ -9,195 +9,195 @@
 -block content
   #pjax
     -block pjax
-    -if show_form
```
Comment (rowanseymour) [inline templates/contacts/contact_import.haml:12]: is indenting this the only change? hard to see from this diff

Comment (norkans7) [reply templates/contacts/contact_import.haml:12]: Yes, when the content fetched on pjax come the replae all the content in `#pjax`

Comment (norkans7) [reply templates/contacts/contact_import.haml:12]: Looking at this again it also looks like we need to switch the `#pjax` and `-block pjax` lines too

Comment (rowanseymour) [reply templates/contacts/contact_import.haml:12]: .. didn't you try this locally?

Comment (norkans7) [reply templates/contacts/contact_import.haml:12]: Yes with `CELERY_ALWAYS_EAGER = FALSE`
RapidPro - Visually build interactive SMS Applications 2020-09-02 17-24-58

**Pull request merged**